examples/storage/cassandra/README.md

```diff
@@ -165,18 +165,18 @@ cassandra   None         <none>        9042/TCP   45s
 
 If an error is returned the service create failed.
 
-## Step 2: Use a Pet Set to create Cassandra Ring
+## Step 2: Use a StatefulSet to create Cassandra Ring
 
-Pet Sets are a new feature that was added as an <i>Alpha</i> component in Kubernetes
+StatefulSets are a new feature that was added as an <i>Alpha</i> component in Kubernetes
 1.3.  Deploying stateful distributed applications, like Cassandra, within a clustered
-environment can be challenging.  We implemented Pet Set to greatly simplify this
-process.  Multiple Pet Set features are used within this example, but is out of
-scope of this documentation.  [Please refer to the Pet Set documentation.](http://kubernetes.io/docs/user-guide/petset/)
+environment can be challenging.  We implemented StatefulSet to greatly simplify this
+process.  Multiple StatefulSet features are used within this example, but is out of
+scope of this documentation.  [Please refer to the StatefulSet documentation.](http://kubernetes.io/docs/user-guide/statefulset/)
```

Comment:
So we need to link the Pet Set docs and add file an issue to update.  Please explain why we are linking the Pet Set docs.